elastalert/alerters/opsgenie.py

```diff
@@ -35,25 +35,29 @@ def __init__(self, *args):
         self.source = self.rule.get('opsgenie_source', 'ElastAlert')
 
     def _parse_responders(self, responders, responder_args, matches, default_responders):
-        if responder_args:
-            formated_responders = list()
-            responders_values = dict((k, lookup_es_key(matches[0], v)) for k, v in responder_args.items())
-            responders_values = dict((k, v) for k, v in responders_values.items() if v)
-
-            for responder in responders:
-                responder = str(responder)
-                try:
-                    formated_responders.append(responder.format(**responders_values))
-                except KeyError as error:
-                    elastalert_logger.warning("OpsGenieAlerter: Cannot create responder for OpsGenie Alert. Key not foud: %s. " % (error))
-            if not formated_responders:
-                elastalert_logger.warning("OpsGenieAlerter: no responders can be formed. Trying the default responder ")
-                if not default_responders:
-                    elastalert_logger.warning("OpsGenieAlerter: default responder not set. Falling back")
-                    formated_responders = responders
-                else:
-                    formated_responders = default_responders
-            responders = formated_responders
+        if responders is None:
+            return None
+        if responder_args is None:
+            responder_args = dict()
```

Comment:
The top two lines are potentially redundant.

The bottom two lines fix a scenario where `responders` contains a placeholder, but `responders_args` isn't configured in a rule at all. Previously, a text with placeholder would end up as a rule. Now, a warning will be shown and `default_responders` will be used instead.